a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -398,3 +398,34 @@ def deallocate_vrid_fip(self, vrid, subnet, amphorae):
             if interface is not None:
                 self._remove_allowed_address_pair_from_port(
                     interface.port_id, vrid.vrid_floating_ip)
+
+    def unplug_aap_port(self, vip, amphora, subnet):
```

Comment:
unplug_vip() will call unplug_aap_port(), and a10_network_tasks.py UnplugVIP() will call it.
So, if we just overwrite this API will we have any side effect?
Can we create another new API (i.e. change the function name) for this?

Reply:
Overwritten the unplug_vip and unplug_aap_port.